From 3d7e0c794b43c0d197ad309cfb0bbf1afc7aa236 Mon Sep 17 00:00:00 2001
From: Baptiste Daroussin <bapt@FreeBSD.org>
Date: Fri, 27 Dec 2024 14:13:03 +0100
Subject: [PATCH] add utils/base64.h to the distribution

fixes: #1353
---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index b0d2b8bfe..8b4d792d6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -22,6 +22,7 @@ EXTRA_DIST= \
 	rak/unordered_vector.h \
 	scripts/checks.m4 \
 	scripts/common.m4 \
-	scripts/attributes.m4
+	scripts/attributes.m4 \
+	utils/base64.h
 
 ACLOCAL_AMFLAGS = -I scripts
